src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2945,7 +2955,7 @@ export class ResultsViewPageStore {
             this.alteredSamples,
             this.unalteredSamples
         ],
-        getSelectedProfile:()=>this.selectedEnrichmentMRNAProfile,// returns an empty array if the selected study doesn't have any mRNA profiles
+        getSelectedProfiles:()=> _.keyBy([this.selectedEnrichmentMRNAProfile],x=>x.studyId),// returns an empty array if the selected study doesn't have any mRNA profiles
```

Comment:
seems kind a weird to use keyBy.  if it's just only one profile, maybe just do with object literal?  { [profile.studyId]:profile }